### PR TITLE
feat(master): support all device orientations on iOS

### PR DESCRIPTION
**Pull request by DaleWebb**
#12

### CODE REVIEW

File: src/controller.js [hunk 1]
```diff
@@ -140,6 +140,11 @@ class Controller extends React.Component {
         transparent
         animationType="none"
         visible={this.state.modals.length > 0}
+        supportedOrientations={[
+          "portrait",
+          "portrait-upside-down",
+          "landscape"
+        ]}
```
Comment (kirkness) [inline src/controller.js:147]: Should this be optional?

**Comment by kirkness** [comment]
Sorry could you also create this off the hooks branch - im yet to merge and release hooks. Will do with this update.

**Comment by DaleWebb** [comment]
The hooks version is #13.

Would you like it to be defined through props and the have the array of all
options as the fallback?

**Comment by DaleWebb** [comment]
https://github.com/teamplanes/react-native-modal-controller/pull/14#issuecomment-545057346: close in favour of #13

**Pull request closed without merging**